src/main/kotlin/org/sol4k/Message.kt

```diff
@@ -0,0 +1,178 @@
+package org.sol4k
+
+import org.sol4k.Constants.PUBLIC_KEY_LENGTH
+import org.sol4k.exception.SerializationException
+import org.sol4k.instruction.CompiledInstruction
+import java.io.ByteArrayOutputStream
+
+data class Message(
```

Comment:
I would propose to rename it to `TransactionMessage` as `Message` is too general. Also, `web3.js` uses `TransactionMessage`; it will be easier for people to understand how to use it

Reply:
done